Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -143,11 +172,11 @@ extension FlowNetwork where Node == PitchSpellingNode.Index, Weight == Double {
     }
 }
 
-extension PitchSpeller.AssignedNode: Comparable {
-    static func < (lhs: PitchSpeller.AssignedNode, rhs: PitchSpeller.AssignedNode) -> Bool {
-        return lhs.index < rhs.index
-    }
-}
+//extension PitchSpeller.AssignedNode: Comparable {
```

Comment:
Shall we burn this?

Reply:
Maybe a "burn" PR after we get through this PR flurry or do you prefer doing it along the way?

Reply:
I'd say lets clear out any dead / commented-out code before merging.